auth-web/src/models/business.ts

```diff
@@ -82,7 +82,6 @@ export interface NameRequest {
     natureOfBusiness?: string
     expirationDate?: Date
     nrNum?: string
-    requestActionCd?: string
```

Comment:
I noticed this was a duplicate of line 81. This probably resulted from a rebase issue at some point. So, I just removed the line.